spec/models/edition_spec.rb

```diff
@@ -86,16 +86,16 @@
         expect(edition.errors.full_messages_for(:state).size).to eq 1
       end
 
-      it "does not allow empty change_note when the update_type is 'major'" do
-        edition = build(:edition, update_type: "major", change_note: "")
+      it "does not allow empty reason_for_change when the update_type is 'major'" do
+        edition = build(:edition, update_type: "major", reason_for_change: "")
         edition.valid?
-        expect(edition.errors.full_messages_for(:change_note)).to eq ["Change note can't be blank"]
+        expect(edition.errors.full_messages_for(:reason_for_change)).to eq ["Reason for change can't be blank"]
       end
 
-      it "allows empty change_note when the update_type is 'minor'" do
-        edition = build(:edition, update_type: "minor", change_note: "")
+      it "allows empty reason_for_change when the update_type is 'minor'" do
```

Comment:
This is just part of the rename but it might be good to test that change_note can be empty on minor too? And probably can't be empty on major too.